src/pybind/mgr/dashboard/frontend/src/app/ceph/smb/smb-tabs/smb-tabs.component.html

```diff
@@ -0,0 +1,54 @@
+<div *ngIf="selectedTab == Tabs.clusters">
+  <h1 i18n>Clusters</h1>
```

Comment:
bit skeptical on the heading as h1. It looks really big. Bigger than what we used to see. Probably we should define the title with a class (hopefully one that is there in the carbon) and then reuse that wherever we want this title to appear.

I saw that we have this format in [nvmeof component](https://github.com/ceph/ceph/blob/main/src/pybind/mgr/dashboard/frontend/src/app/ceph/block/nvmeof-subsystems/nvmeof-subsystems.component.html#L18)

![image](https://github.com/user-attachments/assets/e32ff582-8104-44d5-bb1a-2f3c1a0da102)
this looks a bit more nicer and it takes use of the `cd-help-text` to show the info text which also looks fitting in the page. WDYT?

Reply:
Ah, I wasn't aware we already had something similar in place. Thanks for the heads up, the example you prived looks good, so I'll follow that then.

I've updated it to follow the same pattern.